pipeline/behavior.py

```diff
@@ -0,0 +1,75 @@
+'''
+Schema of session information.
+'''
+import datajoint as dj
+from pipeline import acquisition
+
+schema = dj.schema('ttngu207_behavior',locals())
+
+        
+@schema
+class TrialType(dj.Lookup):
+    definition = """
+    type_name: varchar(64)
+    ---
+    
+    """
+    contents = [
+            ['Hit'],
+            ['Miss'],
+            ['CR'],
+            ['FA'],
+            ['Stimtrials']
+            ]
+
+@schema
+class TrialSet(dj.Manual):
+    definition = """
+    -> acquisition.Session
+    ---
+    number_of_trials: int
```

Comment:
try to go for shorter attribute names - something like `n_trials` here. Give full description of the field in the comment.
```suggestion
    n_trials: int  # total number of trials
```